git-scripts/git-vgrep

```diff
@@ -3,8 +3,22 @@
 set -e
 out=$(mktemp -t git-vgrep-XXXXXX)
 trap "rm -f $out" EXIT
-git grep -n "$@" | tee "$out"
-if [ "$(cat "$out" | wc -l)" -eq 1 ] ; then
+git grep -n "$@" | tee "$out" 1> /dev/null
+
+lines=$(cat -b "$out" | wc -l)
+
+if [ $lines -eq 0 ]; then
+	exit 0
+fi
+
+if [ $lines -eq 1 ] ; then
+	file=$(cut -d: -f1 "$out")
+	line=$(cut -d: -f2 "$out")
+	$EDITOR +"$line" "$file"
+else
+	cat -b "$out"
+	read -p 'Choose line: ' line
+	awk "NR==$line" "$out" | tee "$out"
```

Comment:
Doesn't this clobber `$out` ask `awk` is reading it?

Reply:
I don't know, it was working =)

But I changed to your suggestion to use another intermediary variable.